### ISSUE TITLE
Permission denied when converting Info.plist to binary in GooglePlus.bundle

### ISSUE THREAD

**Issue by achretien** [openbakery/gradle-xcodePlugin#179]
We implemented GooglePlus cocoapod (https://cocoapods.org/pods/googleplus-ios-sdk).

Unfortunately pod unzip the Info.plist contained in `Pods/google-plus-ios-sdk/google-plus-ios-sdk-1.7.1/GooglePlus.bundle` in read-only mode.

Consequently, we get the following error message during the `archive` task :

```
/usr/bin/plutil -convert binary1 /....../GooglePlus.bundle/Info.plist
GooglePlus.bundle/Info.plist: Permission denied
````

I would like to know, why are you converting plist into binary mode and if it is possible to check permission before running the command ? 

Thanks

**Comment by renep** [comment]
I tried to do the same things as xcode do when creating an archive, and on iOS the plists are converted to binary.

I don't think that I need to check the permission, because if the conversion failed, I should ignore this.
I will also add configuration option so that the conversion can be disabled.

**Comment by renep** [comment]
I have added that it should not fail. I plan do release the 0.11.1 bugfix release today, but first me CI muss run all tests ;-)

**Comment by achretien** [comment]
Thanks it should work like this. 
In fact the plist in google API pod is already in binary mode so there is no need to convert  it. 


--------------------
Aurélien Chrétien
CTO @ Recatch.tv
06.47.85.62.66

**Comment by renep** [comment]
I have published a 0.11.1 to repository.openbakery.org, so can you please test this version if it works now.
If so I will also publish it to plugins.gradle.org

**Comment by achretien** [comment]
It’s good, thanks
